### PR TITLE
Bug 1955238: fix images for openstack upgrade jobs

### PR DESCRIPTION
**Pull request by EmilienM**
Fix the image versions for the OpenStack upgrade jobs.

### CODE REVIEW

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Comment by vrutkovs** [comment]
/lgtm cancel

Seems 4.6/4.7/4.8 jobs didn't get `cluster_profile` updated

**Comment by iamemilio** [comment]
These tests are failing, but they are currently failing like this in CI, so the PR is not the issue :)

**Comment by vrutkovs** [comment]
/hold

vexxhost doesn't have Octavia, so upgrade tests would permafail on `timed out waiting for service \"service-test\" to have a load balancer`

**Comment by EmilienM** [comment]
/retest
we now have octavia in our public cloud!

**Comment by vrutkovs** [comment]
Still seeing `timed out waiting for service \"service-test\" to have a load balancer` in periodic-ci-openshift-release-master-ci-4.8-upgrade-from-stable-4.7-e2e-openstack-upgrade, lets give it another try

/retest

**Comment by vrutkovs** [comment]
While we're at that, `openstack-installer` refs need to be fixed in:
* ci-operator/config/openshift/release/openshift-release-master__ci-4.6-upgrade-from-stable-4.5.yaml
* ci-operator/config/openshift/release/openshift-release-master__ci-4.7-upgrade-from-stable-4.7.yaml

Both on these files mistakenly use 4.6 installer, instead of 4.5/4.7

**Review by vrutkovs — CHANGES_REQUESTED** [review]
Needs rebase and updated `openstack-installer` reference to pass upgrades

/lgtm cancel

**Comment by EmilienM** [comment]
4.6 fails to deploy because our OpenStack cloud has self signed certificate. I suggest that we backport the necessary fixes to make it work, if possible.

As for the other failure, it seems related to the platform, Neutron API had some issues to address the query...

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.7-upgrade-from-stable-4.6.yaml [hunk 1]
```diff
@@ -90,13 +90,13 @@ tests:
 - as: e2e-openstack-upgrade
   interval: 48h
   steps:
-    cluster_profile: openstack
+    cluster_profile: openstack-vh-mecha
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.7-upgrade-from-stable-4.6.yaml:93]:
```suggestion
    cluster_profile: openstack-vh-mecha
```

Extra space breaks generated job verification

Comment (EmilienM) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.7-upgrade-from-stable-4.6.yaml]: Thanks!

**Comment by EmilienM** [comment]
/bugzilla refresh
/hold

@vrutkovs what do you think about the [failures](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17197/rehearse-17197-periodic-ci-openshift-release-master-ci-4.8-upgrade-from-stable-4.7-e2e-openstack-upgrade/1419470014100017152), should we add them to a skip list so we make the job green?

**Comment by vrutkovs** [comment]
>@vrutkovs what do you think about the failures

These don't seem to be related to the change

> should we add them to a skip list so we make the job green?

Rehearsal works won't block merging this (`ci/prow/pj-rehearse` is not marked as `Required`)

**Comment by EmilienM** [comment]
> Rehearsal works won't block merging this (`ci/prow/pj-rehearse` is not marked as `Required`)

Right, I guess my question is: "would it be cool to make the job green by excluding some tests that we know fail today, in this PR or maybe in another PR or are we fine with a job that is red all times"

My take: make the job green even if some tests are failing (using TEST_SKIPS)

**Comment by vrutkovs** [comment]
>would it be cool to make the job green by excluding some tests that we know fail today

I don't think its a great idea. Tests failing during upgrade usually indicate bugs, so these should be filed - instead of adding more skipped tests.

**Comment by pierreprinetti** [comment]
@EmilienM Would you please change the message of the commit to reflect what this PR is doing? I will be glad to LGTM then

**Comment by mandre** [comment]
As the bot suggests,
/assign ravisantoshgudimetla

**Comment by mandre** [comment]
BZ https://bugzilla.redhat.com/show_bug.cgi?id=1936874 is no longer relevant and should probably be closed as we removed the 4.5->4.6 upgrade job. Do you mind updating the commit message?

**Comment by pierreprinetti** [comment]
This issue seems to be fixed now: https://github.com/openshift/release/blob/1f9cf8282dd46f692b01f8d1222c38872302fb14/ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.8-upgrade-from-stable-4.7.yaml#L3

/close

@EmilienM please reopen if I am missing something.

**Pull request closed without merging**